fairscale/experimental/__init__.py

```diff
@@ -9,6 +9,7 @@
 
 from typing import List
 
-from . import nn, optim, wgit
+# Don't import sub-modules to avoid experimental stuff gets imported directly
+# when user does an `import fairscale`
```

Comment:
```suggestion
# when user does an `import fairscale`. This can cause experimental code's import
# dependencies (like pygit2) to be leaked into the fairscale main code.
```